cw_mib_generator.py

```diff
@@ -0,0 +1,94 @@
+# @file cwc_mib_generator.py
+#
+# Copyright (C) 2016  Metaswitch Networks Ltd
+
+import re
+import os.path
+
+COMMON_MIB = "./CLEARWATER-MIB-COMMON"
+PC_EXTRAS = "./CLEARWATER-MIB-PC-EXTRAS"
+CWC_EXTRAS = "../CLEARWATER-MIB-CWC-EXTRAS"
+
+PC_MIB_NAME = "./PROJECT-CLEARWATER-MIB"
+CWC_MIB_NAME = "../METASWITCH-CLEARWATER-CORE-MIB"
```

Comment:
I still need to think about, discuss, and finalise the build process. These are easy to change once that happens.